news/12728.feature.rst

```diff
@@ -0,0 +1,5 @@
+``wheel`` is no longer explicitly listed as a build depepndency of ``pip``.
+``setuptools`` already injects this dependency in the ``get_requires_for_build_wheel()`` hook.
+This makes no difference for users of ``pip``.
+This makes no difference when building wheels of ``pip``.
+This avoids an unnecessary dependency on ``wheel`` wehn bulding the source distribution of ``pip``.
```

Comment:
This is what fails in pre-commit:
```suggestion
This avoids an unnecessary dependency on ``wheel`` when building the source distribution of ``pip``.
```

FYI `wheel` has moved into `setuptools` last week during the PyCon US Sprints.

Reply:
Thanks